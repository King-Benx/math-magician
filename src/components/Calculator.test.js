import React from 'react';
import { render, screen } from '@testing-library/react';
import { Calculator } from './Calculator';

test('The calculator renders correctly', () => {
  render(<Calculator />);
  const sampleContent = screen.getByText('Let\'s do some maths!');
  expect(sampleContent).toBeTruthy();
});
