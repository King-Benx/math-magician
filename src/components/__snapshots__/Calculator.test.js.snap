// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The calculator renders fully 1`] = `
Array [
  <div
    className="calc-intro"
  >
    <h1>
      Let's do some maths!
    </h1>
  </div>,
  <div
    className="calculator"
  >
    <div
      className="screen"
    >
      <small />
      <div
        className="display"
        id="display"
      >
        0
      </div>
    </div>
    <div
      className="touch-pad"
    >
      <button
        className=""
        name="AC"
        onClick={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className=""
        name="+/-"
        onClick={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className=""
        name="%"
        onClick={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="operator"
        name="รท"
        onClick={[Function]}
        type="button"
      >
        รท
      </button>
      <button
        className=""
        name="7"
        onClick={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className=""
        name="8"
        onClick={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className=""
        name="9"
        onClick={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="operator"
        name="x"
        onClick={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className=""
        name="4"
        onClick={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className=""
        name="5"
        onClick={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className=""
        name="6"
        onClick={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="operator"
        name="-"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className=""
        name="1"
        onClick={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className=""
        name="2"
        onClick={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className=""
        name="3"
        onClick={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="operator"
        name="+"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className=""
        name="0"
        onClick={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className=""
        name="."
        onClick={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="operator"
        name="="
        onClick={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>,
]
`;
